client/src/components/Login.jsx

```diff
@@ -0,0 +1,185 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+const Container = styled.div`
```

Comment:
Can we move these custom components to another page and import them?
Something similar to how [Kevin set it up](https://github.com/uoftblueprint/tces/tree/kevinle6231/tces-8-urd-dashboard/client/src/components/dashboard-component/nav-dashboard-component) might be nice.
But I would love your opinion on this separation of page construction and component construction.
This would also make it possible to move Login.jsx to a directory in the `pages` directory

Reply:
I can work on refactoring them!

Reply:
Awesome. Thank you! Mega work on this so far

Reply:
Hey Jordan! I refactored my components to follow Kevin's setup. Let me know if anything needs to be changed!